scripts/update-and-check-binary.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+BASE_URL="https://binaries.soliditylang.org/bin"
+REPO_ROOT="$(dirname "$0")/.."
+
+function fail() {
+	echo -e "$@" >&2
+	exit 1
+}
+
+function check_version() {
+	current_version=$(node ./dist/solc.js --version | sed -En 's/^(.*).Emscripten.*/\1/p')
+
+	# Retrieve the correspondent released version
+	short_version=$(echo "${current_version}" | sed -En 's/^([0-9.]+).*\+commit\.[0-9a-f]+.*$/\1/p')
+	release_version=$(curl -s "${BASE_URL}/list.json"  | jq ".releases | .[\"${short_version}\"]" | tr -d '"' | sed -En 's/^soljson-v(.*).js$/\1/p')
```

Comment:
Oh! Nice, I didn't know about the `--raw-output`. Thanks.
Sadly I cannot apply the suggestion because it was marked by git as outdated due to the acceptance of the previous suggestion. But I will add it.

Reply:
Don't worry about git suggestion mechanism :) I actually rarely use the "commit suggestion" option and simply add the change myself in my editor because that's more convenient for me. Most of the time we're going to squash the commits it creates anyway.